components/button/src/Button.js

```diff
@@ -127,7 +129,7 @@ Button.propTypes = {
 	/**
 	 * Button look
 	 */
-	look: PropTypes.oneOf(['primary', 'hero', 'faint', 'link']),
+	look: PropTypes.oneOf(['primary', 'hero', 'faint', 'link', null]),
```

Comment:
with [default prop being `hero`](https://github.com/WestpacGEL/GEL/blob/2b1cf0075fdd665e50da48ced5603c2075aa93c3/components/button/src/Button.js#L236) does this make sense?